source/docs/zero-to-robot/step-2/python-setup.rst

```diff
@@ -0,0 +1,191 @@
+Python Installation Guide
+=========================
+
+This guide is intended for Python teams. Java and C++ teams can skip to :doc:`wpilib-setup`. LabVIEW teams can skip to :doc:`labview-setup`.
+
+Prerequisites
+-------------
+
+You must install a supported version of Python on a supported operating system. We currently support Python 3.8/3.9/3.10/3.11/3.12, but only 3.12 is available for the roboRIO.
+
+Supported Operating Systems and Architectures:
+ * Windows 10 & 11, 64 bit only. 32 bit and Arm are not supported
+ * macOS 12 or higher
+ * Ubuntu 22.04, 64 bit. Other Linux distributions with glibc >= 2.35 may work, but are unsupported
+
+On Windows and macOS, we recommend using the official Python installers distributed by python.org.
+
+* `Python for Windows <https://www.python.org/downloads/windows/>`_
+* `Python for macOS <https://www.python.org/downloads/mac-osx/>`_
+
+Install RobotPy
+---------------
+
+Once you have installed Python, you can use pip to install RobotPy on your development computer.
+
+.. tab-set::
+
+   .. tab-item:: Windows
+      :sync: windows
+
+      .. warning:: On Windows, the `Visual Studio 2019 redistributable <https:// support.microsoft.com/en-us/help/2977003/       the-latest-supported-visual-c-downloads>`_ package is required to be installed.
+
+      Run the following command from cmd or Powershell to install the core RobotPy packages:
+
+      .. code-block:: sh
+
+         py -3 -m pip install robotpy
+
+      To upgrade, you can run this:
+
+      .. code-block:: sh
+
+         py -3 -m pip install --upgrade robotpy
+
+      If you don't have administrative rights on your computer, either use `virtualenv/virtualenvwrapper-win <https://docs.python-guide.org/en/latest/dev/virtualenvs/>`_, or or you can install to the user site-packages directory:
+
+      .. code-block:: sh
+
+         py -3 -m pip install --user robotpy
+
+   .. tab-item:: macOS
+      :sync: macos
+
+      On a macOS system that has pip installed, just run the following command from the Terminal application (may require admin rights):
+
+      .. code-block:: sh
+
+         pip3 install robotpy
```

Comment:
Use `python3 -m pip` for all these. Below, `python3` is used, which may point to a different python than `pip3`